content/docs/arvo/clay/tasks.md

```diff
@@ -4,49 +4,18 @@ weight = 6
 template = "doc.html"
 +++
 
-# Contents
-
-- [Introduction](#introduction)
-- [%warp](#warp) - Read and Subscribe.
-   - [%sing](#sing) - Read a file or directory.
-   - [%next](#next) - Subscribe for the next change to a file or directory.
-   - [%mult](#mult) - Subscribe for the next change to a set of files and/or directories.
-   - [%many](#many) - Track changes to a `desk` for the specified range of revisions.
-   - [Cancel Subscription](#cancel-subscription)
-- [Write and Modify](#write-and-modify)
-   - [%info](#info)
-- [Manage Mounts](#manage-mounts)
-   - [%boat](#boat) - List mounts.
-   - [%mont](#mont) - Mount something.
-   - [%ogre](#ogre) - Unmount something.
-   - [%dirk](#dirk) - Commit changes.
-- [Merge Desks](#merge-desks)
-   - [%merg](#merg)
-- [Permissions](#permissions)
-   - [%perm](#perm) - Set file permissions.
-   - [%cred](#cred) - Add permission group.
-   - [%crew](#crew) - Get permission groups.
-   - [%crow](#crow) - Get group usage.
-- [Foreign Ships](#foreign-ships)
-   - [%warp](#warp-remote) - Read a file on a foreign ship.
-   - [%merg](#merg-remote) - Merge from a foreign `desk`.
-
-# Introduction
-
-This document details all the `task`s you're likely to use to interact with Clay, as well as the `gift`s you'll receive in response. Each section has a corresponding practical example in the [Examples](/docs/arvo/clay/examples) document. Many of the types referenced are detailed in the [Data Types](/docs/arvo/clay/data-types) document. It may also be useful to look at the `++  clay` section of `/sys/lull.hoon` in Arvo where these `task`s, `gift`s and data structures are defined.
+This document details all the `task`s you're likely to use to interact with Clay, as well as the `gift`s you'll receive in response. Each section has a corresponding practical example in the [Examples](/docs/arvo/clay/examples) document. Many of the types referenced are detailed in the [Data Types](/docs/arvo/clay/data-types) document. It may also be useful to look at the `++ clay` section of `/sys/lull.hoon` in Arvo where these `task`s, `gift`s and data structures are defined.
```

Comment:
```suggestion
This document details all the `task`s you're likely to use to interact with Clay, as well as the `gift`s you'll receive in response. Each section has a corresponding practical example in the [Examples](/docs/arvo/clay/examples) document. Many of the types referenced are detailed in the [Data Types](/docs/arvo/clay/data-types) document. It may also be useful to look at the `++  clay` section of `/sys/lull.hoon` in Arvo where these `task`s, `gift`s and data structures are defined.
```